wisp-logging/src/main/kotlin/wisp/logging/Logging.kt

```diff
@@ -4,13 +4,18 @@ import mu.KLogger
 import mu.KotlinLogging
 import org.slf4j.MDC
 import org.slf4j.event.Level
+import wisp.sampling.Sampler
 
 typealias Tag = Pair<String, Any?>
 
 inline fun <reified T> getLogger(): KLogger {
     return KotlinLogging.logger(T::class.qualifiedName!!)
 }
 
+fun KLogger.sampled(sampler: Sampler = Sampler.rateLimiting(100L)): KLogger {
```

Comment:
I just randomly selected `100L` as the default `ratePerSecond` for this.  Happy to take advice on a better/more sensible default.  I could even be convinced that we should provide a default at all and force the user to think about their own requirements.

Reply:
Drop the rate to 1 per second - I think that's the "expected" default rate.  I think it's fine to have a default and to have it low.  If the user wants something different, they'll do the extra step required.